x/dex/types/price.go

```diff
@@ -91,3 +148,34 @@ func ValidateFairOutput(amountIn math.Int, price math_utils.PrecDec) error {
 	}
 	return nil
 }
+
+// Used for generating the precomputedPrice.gob file
```

Comment:
We have `neutrond debug ...` cli commands, maybe it makes sense create an additional CLI debug command in neutron with the logic below in case we need to regenerate `.gob` file?

Reply:
Happy to add it. But seems a bit strange. Neutrond debug seems targeted to end users. There's never a time an end user would need to regenerate the .gob file. The only case would be if we change the maxTickIndex or the tick price logic. I agree it's a bit funny to leave the code commented out. But in my opinion it makes sense as documentation for what the precomputed_prices.gob is and an easy way to change it if we make internal changes. 
Let me know what you think. Happy to change in whatever way makes most sense

Reply:
Yeah, agree. Let's leave it as it as